proposed/2021/DotnetNuGetSign-Verbosity.md

```diff
@@ -0,0 +1,102 @@
+# `dotnet nuget sign --verbosity`
+
+* Status: **In Review**
+* Author: [Heng Liu](https://github.com/heng-liu)
+* Issue: [#11173](https://github.com/NuGet/Home/issues/11173) - No certificate info shows when running dotnet nuget sign on default verbosity
+
+## Problem background
```

Comment:
@kartheekp-ms, https://github.com/NuGet/Home/pull/10466 should be merged with the spec under the implemented/2021 path.

@heng-liu, it would be good to add a reference to @kartheekp-ms's merged spec because this is a similar effort and that spec contains relevant information that you don't want to copy:  the "Log Level Mapping" section.

Reply:
Thanks! Added `Log level mapping` section and it pointed to "[dotnet nuget verify --verbosity](https://github.com/NuGet/Home/blob/dev/implemented/2021/Verify-Verbosity.md#dotnet-nuget-verify---verbosity)".